views/posts/home.handlebars

```diff
@@ -0,0 +1,49 @@
+
+<br>
+<br>
+<br>
+<br>
+<br>
+
+<div class="container text-center">
+  <div class="row">
+    <div class="col-lg-6">
+        <h1 id="hd">Matching Donnors with Needy people in the Society</h1>
+        <br>
+        <br>
+        <br>
+        <div class="row text-center">
+            
+            <div class="col-lg-6 text-center">
+                <h4>For Donnors</h4>
+                <p class="brow">Donation can be made easy by simply uploading an item to the page. Eveyone can donate or become a donnor. </p>
+                <button type="button" class="btn btn-outline-secondary">Start Donating</button>
+            </div>
+            <div class="col-lg-6 text-center">
+                <h4>For Recipient</h4>
+                <p class="brow">This platform is built for anyone who is in need. Are you in need? Time to shop for $0 </p>
+                <button type="button" href="/auth1" class="btn btn-outline-success">Sign Up & Receive</button>
+            </div>
+        </div>
+    </div>
+    <div class="col-lg-6">
+    
+      <img src="hands.jpg" alt="" id="b_w_hands">
```

Comment:
we need to add some data in `alt` attribute here. Also @iamikenna can you please add some fixed height to this image? Without height, image might look very big on some screen.